src/Text/Pandoc/Readers/Textile.hs

```diff
@@ -90,7 +90,10 @@ parseTextile = do
   Pandoc nullMeta . B.toList <$> parseBlocks -- FIXME
 
 noteMarker :: PandocMonad m => TextileParser m Text
-noteMarker = skipMany spaceChar >> string "fn" >> T.pack <$> manyTill digit (char '.')
+noteMarker = do
+  skipMany spaceChar
+  string "fn"
+  T.pack <$> manyTill digit (string "." <|> string "^.")
```

Comment:
Probably you want `try (string "^.")`?  What if the string contains a `^` not followed by `.`?

Reply:
Sorry about that! I'm used to attoparsec's non-consuming `<|>`. I've pushed a commit to wrap in `try`.

I wasn't able to produce any inputs (e.g. `"fn3^ aaa"`) that differed between the `try` and non-`try` versions when parsing a whole doc, but the suggestion expresses the syntax rule more accurately. Thank you!